autogpt_platform/frontend/src/components/node-input-components.tsx

```diff
@@ -445,7 +446,7 @@ const NodeKeyValueInput: FC<{
           </div>
         ))}
         <Button
-          className="w-full"
+          className="w-[183p] rounded-xl bg-gray-200 font-normal text-black hover:text-white"
```

Comment:
This is oddly specific and it uses pixel, will it break responsiveness ? can we use `rem` at least? or instead of setting the width can we set the padding ?

Reply:
W-44 is the closet tailwind class. I should have used that

Reply:
This doesn't change anything I removed it entirely